lib/src/utils/dialog_builder.dart

```diff
@@ -50,16 +50,15 @@ import 'package:ox_coi/src/navigation/navigation.dart';
 
 import 'keyMapping.dart';
 
-showNavigatableDialog({
-  @required BuildContext context,
+showNavigatableDialog({@required BuildContext context,
   @required Widget dialog,
   @required Navigatable navigatable,
   Navigatable previousNavigatable,
-  barrierDismissible = true,
```

Comment:
No real problem, but please let's keep the `, ` at the end of the last item. If we do this adding a new entry at the end only affects one line and everything is easily readable in a diff. If we remove this we always add one `,` in the one line and the content + parenthesis in the other line.

Reply:
I guess it's a habit of Swift development. It's not allowed there. 🙈